processing/app/api/routes/task.py

```diff
@@ -0,0 +1,49 @@
+from io import BytesIO
+
+from app.core import exceptions as api_exceptions
+from app.deps import get_task_service
+from app.models import TaskStatus
+from app.schemas import TaskResponse
+from app.services.task_service import TaskService
+from fastapi import APIRouter
+from fastapi import Depends
+from fastapi.responses import StreamingResponse
+from pydantic import UUID4
+
+router = APIRouter()
+
+
+@router.get(
+    "/{task_id}/status",
+    response_model=TaskResponse,
+)
+async def get_task_status(
```

Comment:
Now this should be called `get_task` and the endpoint should be just `/{task_d}`